src/main/java/mate/academy/onlinebookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,40 @@
 package mate.academy.onlinebookstore;
 
+import java.math.BigDecimal;
+import mate.academy.onlinebookstore.model.Book;
+import mate.academy.onlinebookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    @Autowired
+    private BookService bookService;
```

Comment:
Wrong way to inject bean.
Inject directly via method params in the @Bean method.

```suggestion
```